guides/common/modules/con_transport-modes-for-remote-execution.adoc

```diff
@@ -3,41 +3,43 @@
 
 You can configure your {Project} to use two different modes of transport for remote job execution.
 
+Push-based transport::
 On {SmartProxies} in `ssh` mode, remote execution uses the SSH service to transport job details.
 This is the default transport mode.
 The SSH service must be enabled and active on the target hosts.
 The remote execution {SmartProxy} must have access to the SSH port on the target hosts.
 Unless you have a different setting, the standard SSH port is 22.
 
+Pull-based transport::
+On {SmartProxies} in `pull-mqtt` mode, remote execution uses Message Queueing Telemetry Transport (MQTT) to publish jobs it receives from {ProjectServer}.
+The host subscribes to the MQTT broker on {SmartProxy} for job notifications using the `yggdrasil` pull client.
+After the host receives a notification, it pulls job details from {SmartProxy} over HTTPS, runs the job, and reports results back to {SmartProxy}.
++
+To use the `pull-mqtt` mode, you must enable it on {SmartProxyServer} and configure the pull client on hosts.
+
 [NOTE]
 ====
 If your {SmartProxy} already uses the `pull-mqtt` mode and you want to switch back to the `ssh` mode, run this `{foreman-installer}` command:
 
 [options="nowrap",subs="+quotes,verbatim,attributes"]
 ----
-# {foreman-installer} --foreman-proxy-plugin-remote-execution-script-mode=ssh
+# {foreman-installer} --foreman-proxy-plugin-remote-execution-script-mode ssh
```

Comment:
Why?

Reply:
Personal bias I'm afraid. But it would work, right? :D